src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -273,9 +273,11 @@ public enum RolesMappingResolution {
     public static final String SECURITY_RESTAPI_ADMIN_ENABLED = "plugins.security.restapi.admin.enabled";
     public static final String SECURITY_RESTAPI_ENDPOINTS_DISABLED = "plugins.security.restapi.endpoints_disabled";
     public static final String SECURITY_RESTAPI_PASSWORD_VALIDATION_REGEX = "plugins.security.restapi.password_validation_regex";
+    public static final String SECURITY_RESTAPI_PASSWORD_VALIDATION_REGEX_DEFAULT = "(?=.*[A-Z])(?=.*[^a-zA-Z\\\\d])(?=.*[0-9])(?=.*[a-z]).{8,}";
```

Comment:
I might be using "DEFAULT" incorrectly here, happy to move this back if this isn't the intended use of ConfigConstants.

Reply:
I see, good point. I've removed `ConfigConstants.java` from the change set and moved things back to where they're used.